upload_log_files.py

```diff
@@ -0,0 +1,121 @@
+import argparse
+import os
+from google.cloud import storage
+from urllib.parse import urlparse
+import re
+
+from core_data_modules.logging import Logger
+from storage.google_cloud import google_cloud_utils
+
+from src.lib import PipelineConfiguration
+
+log = Logger(__name__)
+
+
+def fetch_file_paths(dir_path):
```

Comment:
```suggestion
def get_log_file_paths(dir_path):
```
(Fetch implies connection to a remote machine to me)

Reply:
updated thanks